lib/label_printer/label_printer/label.rb

```diff
@@ -24,7 +21,7 @@ def create_labels
       end
 
       def label(asset)
-        { main_label: create_label(asset) }
```

Comment:
This is probably an unnecessary level of abstraction now. I think we'd probably be better of removing it, and either calling `create_label` directly, or alternatively renaming all the `create_label` implementations to `label`. The later is more work, but I think is a better method name. (As we're not really creating anything)

Reply:
I've removed the `label` method, and renamed `create_label` to `build_label`